godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -152,14 +155,13 @@ class TractViewController: BaseViewController {
     
     // MARK: - Segmented Control
     
-    fileprivate func languageSegmentedControl() -> UISegmentedControl {
+    fileprivate func configureLanguageSegmentedControl() {
         let primaryLabel = self.determinePrimaryLabel()
         let parallelLabel = self.determineParallelLabel()
         
-        let segmentedControl = UISegmentedControl(items: [primaryLabel, parallelLabel])
-        segmentedControl.selectedSegmentIndex = 0
-        segmentedControl.addTarget(self, action: #selector(didSelectLanguage), for: .valueChanged)
-        return segmentedControl
+        languageSegmentedControl = UISegmentedControl(items: [primaryLabel, parallelLabel])
+        languageSegmentedControl!.selectedSegmentIndex = 0
```

Comment:
The weak that i put there was really bad actually. since that was a weak reference, it could get dropped immediately after creating the object before it gets added as a subview. 👎